infra/nnfw/command/configure

```diff
@@ -2,25 +2,5 @@
 
 import "build.configuration"
 
-INSTALL_PATH="${NNFW_INSTALL_PREFIX:-${WORKSPACE_PATH}/out}"
-# Normalize to absolute path
-if [[ "${INSTALL_PATH}" != /*  ]]; then
-    INSTALL_PATH=${NNFW_PROJECT_PATH}/${INSTALL_PATH}
-fi
-
-# Create "BUILD_PATH"
-mkdir -p "${BUILD_PATH}"
-
-if [ ! -d "${INSTALL_PATH}" ]; then
-  echo "[WARNING] Cannot find install directory '${INSTALL_PATH}'"
-  echo "          Try to make install directory"
-  mkdir -p "${INSTALL_PATH}"
-  if [ ! -d "${INSTALL_PATH}" ]; then
-    echo "[ERROR] Fail to make install directory '${INSTALL_PATH}'"
-    echo "        Please make '${INSTALL_PATH}' directory first"
-    exit 255
-  fi
-fi
-
-cd "${BUILD_PATH}"
-cmake "${NNFW_PROJECT_PATH}"/infra/nnfw -DCMAKE_INSTALL_PREFIX="${INSTALL_PATH}" "$@"
+# Require cmake version >= 3.13
```

Comment:
Don't we need version check? Although, required version is very old, it is better to notify the user with specific version required rather quitting with kind of 'Unsuppoted option' error.

Reply:
It may be helpful to get the cmake version only (wo/ banner) with following command.
```
cmake -E capabilities |  grep -o '"string":"[^"]*' | grep -o '[^"]*$'
```

Reply:
Ah yes, but cmake command will fail before the cmake_minimum_requred() macro.
My worries was about lacking information of failure. If cmake version is older than required version, error message would be something like this.

```
$cmake -S ./infra/nnfw
CMake Error: Unknown argument -S
CMake Error: Run 'cmake --help' for all supported options.
```

It would be nice if it just tells checking the version of cmake.
```
$cmake -S ./infra/nnfw
[ Error ] Your cmake version (x.x.x) is too old, minimum required version is 3.16.3.
```

However, this is just my preference, optional opinion. Don't mind it ;)

Reply:
I tried to add error message when cmake version is old. But I decided to not add message because it is not simple to check cmake version on bash script working both on ubuntu and tizen.